versionist.conf.js

```diff
@@ -1,32 +1,59 @@
-module.exports = {
+var _ = require('lodash');
+var execSync = require('child_process').execSync;
 
-  subjectParser: 'angular',
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
```

Comment:
Much like you did with `lowerFooterTags`, would it be worth including this as a standard option within versionist itself, which you could turn on or off with a flag in `versionist.conf.js`?